hpe3parclient/client.py

```diff
@@ -4816,3 +4816,125 @@ def getScheduleStatus(self, schedule_name):
                 return 'active'
         msg = "Couldn't find the schedule '%s' status" % schedule_name
         raise exceptions.SSHException(reason=msg)
+
+    @staticmethod
+    def convert_cli_output_to_wsapi_format(cli_output):
```

Comment:
@ansarars Please call this static method in _convert_cli_output_to_collection_like_wsapi() to avoid code redundancy.